.pre-commit-config.yaml

```diff
@@ -1,21 +1,23 @@
+# .pre-commit-config.yaml
+---
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
-    hooks:
-      - id: end-of-file-fixer
-      - id: trailing-whitespace
-      - id: check-yaml
-        exclude: ^base/jenkins/templates/.*
-      - id: check-added-large-files
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
```

Comment:
You should be using frozen pre-commit hooks. Do the following:

`pre-commit autoupdate --freeze`